### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by yash9263**
Add alert styles and documentation

### CODE REVIEW

**Review by partha8 — COMMENTED** [review]
Looks good!

File: docs.html [hunk 1]
```diff
@@ -36,7 +38,7 @@
           <a href="#avatar">
```
Comment (vrx29) [inline docs.html:38]: anchor tag should be inside li tag. You have put li tag inside anchor tag.

**Review by shahbaz4595 — APPROVED** [review]
Looks good

File: docs.html (hunk 1 above)
Comment (yash9263) [reply docs.html:38]: Thanks, Vineet will fix it.

**Comment by yash9263** [comment]
Thanks @partha8, @vrx29 and @shahbaz4595 for the review.

**Pull request merged**